### DISCUSSION TITLE
Where to file bugs?

### DISCUSSION THREAD

**Discussion by hikhvar** [PaperWM-community/PaperWM#11] [General]
Hey, 

where should we file bugs with the next-release branch of this repository? As the maintainer of https://aur.archlinux.org/packages/gnome-shell-extension-paperwm-git/, I tried the next-release as of today. I can't scroll my windows left and right with <SUPER>/Left or <SUPER>/Right. Only <SUPER>. and <SUPER>, do work.

**Comment by michaelhthomas** [comment]
> Hey, 
> 
> where should we file bugs with the next-release branch of this repository? As the maintainer of https://aur.archlinux.org/packages/gnome-shell-extension-paperwm-git/, I tried the next-release as of today. I can't scroll my windows left and right with <SUPER>/Left or <SUPER>/Right. Only <SUPER>. and <SUPER>, do work.

Try removing the Gnome shortcuts for those same key combinations, I was having some weird conflicts as well.

**Reply by hikhvar** [reply]
This works! Thank you. Ok, I was on the upstream next-release branch before. Hadn't this issue yet.

**Reply by michaelhthomas** [reply]
I had the same experience, so I'm pretty sure it is something related to the changes made in this branch. Just not sure exactly what yet!

**Comment by cstich** [comment]
I can't use my keyboard to select workspaces with the current state of `next-release`. I have defined `Super+1`, `Super+2`, etc. to switch to the corresponding workspaces in `gnome` but this does not currently work while the extension is activated.

**Reply by michaelhthomas** [reply]
Do those shortcuts work on the upstream next-release branch? If so, it might be an issue with the updated settings.

**Reply by cstich** [reply]
Yes, they do work upstream on the next-release branch.

**Reply by mogenson** [reply]
The GTK4 Settings UI from this next-release branch grabs the Super key and blocks all other Gnome Shell keybindings that use Super :(
